File: tests/fixtures/sample.swift
import Foundation
import UIKit

public protocol Processable {
    func process(input: String) -> String
}

public class DataManager: NSObject, Processable {
    private var count: Int = 0
    public let name: String = ""

    public func process(input: String) -> String {
        return input.uppercased()
    }

    private func helper() {
    }
}

public func createManager(name: String) -> DataManager {
    return DataManager()
}

File: tests/fixtures/sample.c
#include <stdio.h>
#include "myheader.h"

static int internal_counter = 0;
int global_value = 42;

static void helper(void) {
    printf("helper\n");
}

int process(const char *input, int length) {
    helper();
    printf("%s\n", input);
    return length;
}
